configs/ip_allow.yaml.default

```diff
@@ -25,7 +25,7 @@
 # That means you generally want to append your rules after the ones listed here.
```

Comment:
ip_addr_acl referenced above this block

Reply:
Let me check, I thought I found all those references.